dockers/docker-database/base_image_files/monit_database

```diff
@@ -0,0 +1,7 @@
+###############################################################################
+## Monit configuration for database container
+## process list:
+##	redis_server
+###############################################################################
+check process redis_server matching "/usr/bin/redis-server"
+	if does not exist then alert
```

Comment:
Monit will start before any of the containers, therefore this line will cause monit to generate false alarm alerts until the containers have started. Please look into adding syntax like we use for other monit alerts (e.g., `if does not exist for X times within Y cycles then alert`). If this syntax works, then modify all similar lines in this file to read `if does not exist for 5 times within 5 cycles then alert` and open a separate PR which changes the global monit interval in monitrc to 60 seconds. This way, monit will only alert if a process is not running for 5 minutes.

Reply:
I will investigate and change all the lines in this PR if it does work. Also submit a separate PR to change the global monit interval to 60 seconds.

Reply:
@jleveque The syntax like _**if does not exist for 5 times within 5 cycles then alert**_ does work in such scenario. I will do that according to our discussion.